eagles-food-backend/Program.cs

```diff
@@ -18,7 +18,7 @@
 
 var builder = WebApplication.CreateBuilder(args);
 
-var connectionString = builder.Configuration.GetConnectionString("DefaultConnection");
+var connectionString = builder.Configuration.GetConnectionString("TestConnection");
```

Comment:
nope, DefaultConnection is the way, you want to run `git update-index --assume-unchanged Program.cs` i think

Reply:
That was for my personal test and anyone in the team that does not have MySQL.
Hence the name TestConnection.